src/components/CompletedItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { RiCloseCircleLine } from 'react-icons/ri';
+
+function CompletedItem({ singleToDo, updateToDo }) {
+  return (
+    <div
+      className={singleToDo.completed ? 'todo-row complete' : 'todo-row'}
+      style={singleToDo.important ? { background: 'orange' } : {}}
```

Comment:
what is `important` here?

Doesn't look this will ever equate to true

Reply:
Updated.